requirements.txt

```diff
@@ -18,3 +18,6 @@ sortedcontainers==2.1.0
 wcwidth==0.1.9
 websocket-client==0.57.0
 zipp==3.1.0
+pytz==2020.1
```

Comment:
do you need to add anything to setup.py? 

guess its only essential packages that are there, not stuff that'ss in examples?

Reply:
ye that makes sense